index.html

```diff
@@ -0,0 +1,229 @@
+                <p class="program-description">Appreciate various creations applying  CC license of artists, organized from Art Center Nabi.</p>
+            </div>
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/chat.png" alt="Forum-Icon">
+                <p class="program-title fw-bold fs-6 py-1">Forum</p>
+                <p class="program-description">Have the time to share your thoughts and opinions with experts for each topic.</p>
+            </div>
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/presentation.png" alt="Workshop-Icon">
+                <p class="program-title fw-bold fs-6 py-1">Workshop</p>
+                <p class="program-description">Try creating your own work using open source license rather than just watching.</p>
+            </div>
+            <div class="dt-course-detail d-flex flex-column align-items-center">
+                <img class="program-image" src="images/connection.png" alt="Network-Icon">
+                <p class="program-title fw-bold fs-6 py-1">CC Ignite</p>
+                <p class="program-description">Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+            </div>
+        </div>
+        <a href="#" class="courses-btn">See All Courses</a>
+    </section>
+    <!-- Featured Speakers -->
+    <section id="speakers" class="d-flex flex-column align-items-center px-2 py-4">
+        
+        <h3 class="fw-bold mb-5">Featured Instructors</h3>
+
+        <div class="instructors-list">
+            <!-- check javascript for dynamic data -->
+        </div>
+
+        <div class="card w-100 text-center mt-3 more-btn">
+            <div class="card-body">
+              MORE <img src="images/down-arrow.png" alt="More-Options-Icon">
+            </div>
+        </div>
+
+    </section>
+    <!-- footer -->
+    <section id="partner" class="partner-section p-4">
+        <div class="partners-group">
+            <div class="partner-heading">Partner</div>
+            <div class="partners">
+                <p class="partners-list">mozilla</p>
+                <p class="partners-list">Google</p>
+                <p class="partners-list">NAVER</p>
+                <p class="partners-list">daumkakao</p>
+                <p class="partners-list">airbnb</p>
+            </div>
+        </div>
+    </section>
+    <section id="footer" class="p-2">
+        <div class="container-fluid">
+            <div class="row p-4">
+                <div class="col-6 text-center p-0 pe-2">
+                    <img src="images/creative-commons.png" alt="Creative-Commons-Korea-Logo" width="150px" height="35px">
+                </div>
+                <div class="col-6 p-0 ps-4 footer-description">
+                    2015 Creative Commons Korea. <br> Some Rights Reserved.
```

Comment:
- **[OPTIONAL]** I suggest to personalize this section with the topic of your page, we could use our logo, the current year, and our country. 👇🏽 

```
2023 Web development [Your country]. 
Some Rights Reserved
```